src/mpid/ch4/shm/src/shm_p2p.h

```diff
@@ -21,7 +21,7 @@
     - name        : MPIR_CVAR_CH4_XPMEM_LMT_MSG_SIZE
       category    : CH4
       type        : int
-      default     : 4096
+      default     : -1
       class       : device
```

Comment:
I'd prefer to enable xpmem by default. Most users may expect this feature without extra environment variable setting.

Reply:
Also fine. I can drop that patch.